src/view.js

```diff
@@ -56,7 +56,7 @@ const View = Backbone.View.extend({
   // it exists, for the template
   serializeData() {
     if (!this.model && !this.collection) {
-      return {};
+      return;
```

Comment:
Is this check strictly needed at all? Could it be removed so `serializeData` becomes:

```js
serializeData() {
  if (this.model) {
    return this.serializeModel()
  }
  if (this.collection) {
    return { items: this.serializeCollection() };
  }
}
```

It cuts a couple lines and removes a redundant check for `this.model`.